CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 ## [Unreleased]
 ### Changed
  - Add option to explicitly include specific namespaces (@simulalex)
+### Removed
```

Comment:
`### Breaking Change`. We use removed when we safely remove something but it does not require any action from a user to upgrade. In this case if they are on ruby 2.0.0 (not sure why they would be but I have heard of some very odd setups) they would need to upgrade their ruby version to continue working so we want the change very loud and obvious. I will make a major bump on release.